fearless/Modules/Staking/StakingMain/StakingMainPresenter.swift

```diff
@@ -510,6 +510,16 @@ extension StakingMainPresenter: StakingMainInteractorOutputProtocol {
         }
     }
 
+    func didReceive(eraCompletionTimeResult: Result<UInt64, Error>) {
+        switch eraCompletionTimeResult {
+        case let .success(time):
+            let eraCompletionTimeInSeconds = TimeInterval(time / 1000)
```

Comment:
We have TimeInterval extension that contains conversions (milliseconds, seconds, dayFromSeconds)